plugins/bbcode/plugin.js

```diff
@@ -660,13 +666,18 @@
 								tagName = 'color';
 								value = CKEDITOR.tools.convertRgbToHex( value );
 							} else if ( ( value = style[ 'font-size' ] ) ) {
+                                                                tagName = 'size';
 								var percentValue = value.match( /(\d+)%$/ );
 								if ( percentValue ) {
 									value = percentValue[ 1 ];
-									tagName = 'size';
```

Comment:
I don't see a reason to move this assignment, in fact moving it will lead to quirky cases, where tag is inserted, but no value is going to be assigned.